test/unit/flushPromises.js

```diff
@@ -0,0 +1,16 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+/**
+ * Returns a promise that will be resolved after all currently resolved promises.
+ * @returns {Promise}
+ */
+export default () => new Promise(resolve => setTimeout(resolve));
```

Comment:
If we wanted to be consistent with what's being done in the `test/functional` directory, we would have:

`test/unit/helpers/flushPromises.js`
`test/unit/specs/...` <-- all specs would be moved down to this directory.

Thoughts?

Reply:
I like it.  I think there are more helpers we could extract.

Reply:
Cool, I'll propose this change in a separate PR.